Monikor-End displaying on rendered page

DominicJ2 opened the pull request:
The `::: Monikor-End` tag is displaying on the [rendered page for this](https://docs.microsoft.com/en-us/azure/devops/organizations/settings/work/customize-process?view=vsts&tabs=new-nav)  I'm not what the reason is, but the only difference I see between this page and other pages that do not display `::: moniker-end` is an additional new line between the content and the `::: moniker-end` tag.

This change removes the additional new line.

KathrynEE commented:
@Dominic12 - Thank you for this contribution, however, I updated this file yesterday and removed the extra moniker tag. So am closing this.

DominicJ2 commented:
> @Dominic12 - Thank you for this contribution, however, I updated this file yesterday and removed the extra moniker tag. So am closing this.

@KathrynEE [I was looking at your change](https://github.com/MicrosoftDocs/vsts-docs/commit/c35b56a424157764a0e0bbd7c6d0a6832707b48c#diff-99b392b6d5eb758986ceb84ca6b7a981), and I can't tell what caused/fixed the issue. Can you help shed some light on it, because I see it happening on a few other pages?
EX1: https://docs.microsoft.com/en-us/azure/devops/project/feedback/?view=vsts
EX2: https://docs.microsoft.com/ru-ru/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-2.1&tabs=aspnetcore2x
EX3: https://docs.microsoft.com/ja-jp/azure/devops/project/feedback/?view=vsts
